doc/tutorials/vuart_configuration.rst

```diff
@@ -267,7 +267,7 @@ and :ref:`How to Configure a Communication Port
 <how-to-configure-a-communication-port>`.
 
 You can configure both Legacy vUART and PCI-vUART in
-``./misc/vm_configs/xmls/config-xmls/<board>/<scenario>.xml``. For
+``./misc/config_tools/data/<board>/<scenario>.xml``. For
```

Comment:
A more generic comment here. We should really advice people not to make direct changes to these files. Instead, they should create a copy, make modifications to the copy and then use it with the `SCENARIO` parameter.

Reply:
yes indeed.  We don't want to have folks editing files that are under git control unless they intend to submit a PR

Reply:
In the third patch of this PR, I've updated the instructions in ``vuart_configuration.rst`` and ``enable_ivshmem.rst`` a bit so that they no longer instruct users to modify predefined scenario XMLs.